PhysicsTools/PyTorch/interface/TorchCompat.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PhysicsTools_PyTorch_interface_TorchCompat_h
+#define PhysicsTools_PyTorch_interface_TorchCompat_h
+
+// TODO: find a better way to resolve TorchLib and ROOT's ClassDef macro clash.
+// See: https://root-forum.cern.ch/t/use-of-torch-model-inside-root-dataframe-class-functor/62797
+#ifdef ClassDef
+#undef ClassDef
+#endif
```

Comment:
Would the `pragma`-based suggestion by Danilo work here ?
I'm somewhat concerned that some package may include this header (indirectly, maybe) and later try to use ROOT's `ClassDef` macro.